cewl.rb

```diff
@@ -982,6 +987,11 @@ def usage
 								words.gsub!(/[^[[:alpha:]]]/i, " ")
 							end
 
+							if convert_umlauts then
+								words.gsub!(/[^a-zäöüßÄÖÜ]/i, " ")
```

Comment:
Why are you doing this? It will lose characters like ë, è, é, and ê

Reply:
You're right - This line has to be removed. (This code is a remnant of another use case in which I had to replace the most common umlauts and remove all the remaining/other ones.)